src/observable/throw.ts

```diff
@@ -4,7 +4,7 @@ import {Subscription} from '../Subscription';
 
 export class ErrorObservable<T> extends Observable<T> {
 
-  static create<T>(error: T, scheduler?: Scheduler) {
+  static create<T>(error: any, scheduler?: Scheduler) {
```

Comment:
Actually, I'm have tried the way to put generic type in class, it seems
```typescript
Observable<Item>.method();
```
is illegal anyway, as I'm from C# it somehow confused me.